1907-05-13.xml

```diff
@@ -0,0 +1,1805 @@
+                    </div>
+                    <div type="advert" xml:id="deg-ad-phc01">
+                        <head>P. HENDERSON &amp; CO’s LINE.</head>
+                        <p>Steamers leave SUEZ and PORT SAID fortnightly for LONDON or LIVERPOOL
+                            direct.</p>
+                        <p>(Electric Light.) SALOON (Amidships) FARE £12. (Latest improvements.)</p>
+                        <p><name>S.S. RANGOON</name>
+                            <measure quantity="6000" unit="ton">6000</measure> Tons will leave PORT
+                            SAID about <date when="1907-05-16">May 16</date> for London.</p>
+                        <p><name>S.S. BURMA</name>
+                            <measure quantity="5600" unit="ton">5600</measure> Tons will leave PORT
+                            SAID about <date when="1907-05-24">May 24</date> for London.</p>
+                        <p><name>S.S. ARRACAN</name>
+                            <measure quantity="5800" unit="ton">5800</measure> Tons will leave PORT
+                            SAID about <date when="1907-05-25">May 25</date> for Liverpool</p>
+                        <p>Due in LONDON or LIVERPOOL 12 days thereafter.</p>
+                        <p>Apply WORMS &amp; Co., Port Said and Suez. THOS. COOK &amp; SON, (EGYPT)
+                            LD., CAIRO ;</p>
+                        <p>G. J. GRACE &amp; CO., ALEXANDRIA.</p>
+                    </div>
+                    <div type="advert" xml:id="deg-ad-ens01">
+                        <head>EXPRESS NILE STEAMER Co.</head>
+                        <p>Cairo-Luxor Tourist Service -- Luxor-Assouan Express Service.</p>
+                        <p>Quick Freight Service ALEXANDRIA-CAIRO.</p>
+                        <p><measure>89662</measure> For Bookings and Particulars apply: CAIRO
+                            OFFICES, 19, CHAREH-EL MADABEGH.</p>
+                    </div>
+                    <div type="advert" xml:id="deg-ad-cun03">
+                        <head>Cunard Line.</head>
+                        <p><hi rend="bold">Royal Mail Steamers</hi> from Liverpool to New York and
+                            Boston every Saturday and Tuesday.</p>
+                        <p><hi rend="bold">Regular Twin Screw Sailings</hi> at short intervals from
+                            Trieste, Fiume and Naples to New-York.</p>
+                        <p>Through tickets from Egypt delivered by <hi rend="bold">Messrs.
+                                Rodacanachi &amp; Co.,</hi> Alexandria, <hi rend="italic">General
+                                Agents</hi> ; <hi rend="bold">Nicolas Kirzis</hi>, Cairo; <hi
+                                rend="bold">R. Broadbent</hi>, Port Said, <hi rend="italic"
+                                >Agents</hi>. <measure type="indexNo">19-1-907</measure></p>
+                    </div>
+                </div>
```

Comment:
move this `</div>` tag, which closes the section, to the end of the page (just before the `</div>` that ends page 1).